### PR TITLE
Add social card config options

### PR DESCRIPTION
**Pull request by SaraVieira**
- Adds ability to pass a meta share image
- Ability to pass an alt to that image
- Ability to pass twitter id

related https://github.com/axodotdev/oranda/issues/24

### CODE REVIEW

File: oranda.json [hunk 1]
```diff
@@ -2,5 +2,10 @@
   "theme": "axo",
   "homepage": "https://oranda.axo.dev",
   "repository": "https://github.com/axodotdev/oranda",
-  "additional_pages": ["./docs/src/theme/themes.md"]
+  "additional_pages": ["./docs/src/theme/themes.md"],
+  "social": {
+    "image": "https://www.axo.dev/meta_small.jpeg",
+    "image_alt": "AXO",
```
Comment (ashleygwilliams) [inline oranda.json]: 😱 all caps?? lol

File: src/site/html.rs [hunk 2]
```diff
@@ -2,6 +2,37 @@ use axohtml::{dom::DOMTree, html, text, unsafe_text};
 
 use crate::config::{theme, Config};
 use axohtml::elements::div;
+use axohtml::elements::meta;
+
+// Clippy issue as explained here
```
Comment (ashleygwilliams) [inline src/site/html.rs]: I might say "false positive duplicate allocation warning" and link to this: https://github.com/rust-lang/rust-clippy/issues?q=is%3Aissue+redundant_allocation, which will be more inclusive of the types of errors (that issue is not *exactly* the one we have, but similar)

**Review by ashleygwilliams — APPROVED** [review]
would like a few *extremely* small changes but i think this is good to go otherwise!

File: src/site/html.rs (hunk 2 above)
Comment (SaraVieira) [reply src/site/html.rs]: Done!

File: oranda.json (hunk 1 above)
Comment (SaraVieira) [reply oranda.json]: also fixed lol

**Pull request merged**